chain4energy/assetlist.json

```diff
@@ -21,7 +21,7 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chain4energy/images/c4e.png"
       },
-      "coingecko_id": "",
+      "coingecko_id": "chain4energy",
```

Comment:
The cgid isn't in CoinGecko's API.
https://api.coingecko.com/api/v3/coins/list

Reply:
https://www.coingecko.com/en/coins/chain4energy